app/form/v1_0/fields/drug-use/add-drugs.ts

```diff
@@ -69,21 +87,33 @@ const otherDrugNameField: FormWizard.Field = {
   labelClasses: utils.getSmallLabelClassFor(FieldType.Text),
 }
 
-const drugUsageAmphetamines = misusedDrugsLastUsedField('AMPHETAMINES')
-const drugUsageBenzodiazepines = misusedDrugsLastUsedField('BENZODIAZEPINES')
-const drugUsageCannabis = misusedDrugsLastUsedField('CANNABIS')
-const drugUsageCocaine = misusedDrugsLastUsedField('COCAINE')
-const drugUsageCrack = misusedDrugsLastUsedField('CRACK')
-const drugUsageEcstasy = misusedDrugsLastUsedField('ECSTASY')
-const drugUsageHallucinogenics = misusedDrugsLastUsedField('HALLUCINOGENICS')
-const drugUsageHeroin = misusedDrugsLastUsedField('HEROIN')
-const drugUsageMethadoneNotPrescribed = misusedDrugsLastUsedField('METHADONE_NOT_PRESCRIBED')
-const drugUsageMisusedPrescribedDrugs = misusedDrugsLastUsedField('MISUSED_PRESCRIBED_DRUGS')
-const drugUsageOtherOpiates = misusedDrugsLastUsedField('OTHER_OPIATES')
-const drugUsageSolvents = misusedDrugsLastUsedField('SOLVENTS')
-const drugUsageSteroids = misusedDrugsLastUsedField('STEROIDS')
-const drugUsageSpice = misusedDrugsLastUsedField('SPICE')
-const drugUsageOtherDrug = misusedDrugsLastUsedField('OTHER_DRUG')
+const drugUsageAmphetamines = misusedDrugsLastUsedField(drugsList.AMPHETAMINES.value, drugsList.AMPHETAMINES.text)
```

Comment:
I think you can use the `drugsList` from above to simplify this whole section like
```
const mapDrugUsage = Object.values(drugsList).map(drug =>
  misusedDrugsLastUsedField(drug.value, drug.text)
)

const [ drugUsage1Name, drugUsage2Name] = mapDrugUsage
```

Reply:
resolved in [41367c8](https://github.com/ministryofjustice/hmpps-strengths-based-needs-assessments-ui/pull/803/commits/41367c83db19d150e9d02fe7827999f1c7f1e228)